### ISSUE TITLE
make citeable in ZENODO

### ISSUE THREAD

**Issue by tischi** [fiji/Correct_3D_Drift#4]
@ctrueden @acardona @imagejan @robertb-r 
Someone approached me about how to cite the Correct_3D_Drift plugin...

I could make it cite-able via ZENODO. I did that already for a couple of other plugins:
https://zenodo.org/deposit?page=1&size=20
What do you think?
Of course we could use another (shared) ZENODO account for this rather than my private one.

**Comment by tischi** [comment]
....I just found out that there is a publication connected to the code here :-)

Parslow A, Cardona A and Bryson-Richardson RJ (2014) Sample drift correction following 4D confocal time-lapse imaging. J Vis Exp. 2014 Apr 12;(86). doi: 10.3791/51086.

That publication should of course be cited in any case. However, since I took over the maintenance of the plugin I added couple of new features. Question is whether we want people still just to cite the original publication or maybe (in addition), e.g. a ZENODO link pointing to the latest version. Or something else? I am very open to any kind of modality!

**Comment by acardona** [comment]
Hi @tischi, in a scientific publication likely the best is to refer to the paper by Parslow et al. Although the plugin was originally written pretty much entirely by me, during the course of Gabriel Martin's EMBO imaging workshop in Oeiras, Portugal. Citing the code directly would be fine too, by referring to its web page or github repository.

**Comment by robertb-r** [comment]
Hi @acardona and @tischi, happy with any of the suggested approaches. The aim of the publication was to demonstrate the use of the plugin, but as @acardona said, he certainly wrote most of it in the first place (although I thought I did manage more than (1-pretty much entirely)!) and I appreciate there have been many improvement to the plugin since that time that should be acknowledged. Referring to gitub has the advantage that it will stay current with future additions/contributors, where as ZENODO, is a snapshot that will also go out of date (if I understand correctly).

**Comment by acardona** [comment]
I agree with Robert, referring to github's repository is the most future-proof: up to date, and full version history with authorships.

**Comment by tischi** [comment]
In ZENODO, you get in fact two DOIs, one pointing to a specific release and one pointing always to the latest release. I typically use the latter one, so it would not go out of date:

https://help.zenodo.org/#versioning

We could of course also try to directly cite the github repo, but I feel that ZENODO is the more recommended way of achieving this...I am now frankly also a bit confused why that is, but maybe having a DOI is something good? @haesleinhuepf @fjug

https://genr.eu/wp/cite/

**Comment by robertb-r** [comment]
It looks like a good option then, just one I was not aware of. Happy with whichever approach you take, and thanks for all o the improvement you have made.

**Comment by haesleinhuepf** [comment]
Hi @tischi ,

a colleague of mine @StephanJanosch wrote a nice blog entry touching the topic:
https://www.de-rse.org/blog/howto/2017/05/08/how-to-assign-a-doi-to-software-within-mpg.html

The first part is about MPG-style creating DOIs, the second part explains a bit how and why we should be doing this.

I hope this helps.

Cheers,
Robert
